Header header
float32 drone_timeuint16 tag
uint16 size
uint32 double_tap_counter
uint32 finish_line_counter
